test/Mono.Linker.Tests.Cases/RequiresCapability/Dependencies/RequiresAttributeInCopyAssembly.cs

```diff
@@ -9,48 +9,56 @@
 
 namespace Mono.Linker.Tests.Cases.RequiresCapability.Dependencies
 {
-	public class RequiresUnreferencedCodeInCopyAssembly
+	public class RequiresAttributeInCopyAssembly
 	{
-		public RequiresUnreferencedCodeInCopyAssembly ()
+		public RequiresAttributeInCopyAssembly ()
 		{
 		}
 
 		[RequiresUnreferencedCode ("Message for --Method--")]
+		[RequiresAssemblyFiles (Message = "Message for --Method--")]
 		public void Method ()
 		{
 		}
 
 		[RequiresUnreferencedCode ("Message for --UncalledMethod--")]
+		[RequiresAssemblyFiles (Message = "Message for --UncalledMethod--")]
```

Comment:
This looks fragile - in that if somebody adds a new test case and forgets to add both attributes we're loosing coverage (and it's really hard to find). Maybe we can "fix" this by:
* Adding a unit test which goes over the test classes in the right namespace and validates that if one attribute is there, there's also the second one
* Would need some way to tell this test that there are cases where it's intentional that there's a difference.